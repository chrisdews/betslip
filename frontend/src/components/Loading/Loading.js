import React from "react";
import styled from "styled-components";
import oclogo from "../../images/oc-logo.png";

const Loading = ({ children }) => {
  return (
    <LoadingStyle>
      <h3>{children}</h3>
      <Logo src={oclogo} />
    </LoadingStyle>
  );
};

export default Loading;

const Logo = styled.img`
  width: 107px;
  height: 20px;
`;

const LoadingStyle = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  flex-direction: column;
  margin: auto;
  margin-top: 30vh;
`;
