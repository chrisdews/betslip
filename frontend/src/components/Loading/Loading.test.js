import React from "react";
import Loading from "./Loading";
import { render } from "@testing-library/react";

describe("Loading", () => {
  it("renders correctly - snapshot", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});