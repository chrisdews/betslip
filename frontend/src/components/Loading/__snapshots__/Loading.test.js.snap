// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly - snapshot 1`] = `
<div>
  <div
    class="sc-gtsrHT jBxogo"
  >
    <h3 />
    <img
      class="sc-bdnxRM kmtgCg"
      src="oc-logo.png"
    />
  </div>
</div>
`;
